source/iface/command-accessor.rst

```diff
@@ -6,281 +6,274 @@
+Available only when
+``(AccessMode != access_mode::atomic &&
+AccessMode != access_mode::read && Dimensions == 0)``.
 
-  *Reference variants*
-  dataT &operator[](size_t index) const;
-  dataT &operator[](sycl::id<dimensions> index) const;
+Assignment to the single element that is accessed by this accessor.
 
-  *Value variants*
-  dataT operator[](size_t index) const;
-  dataT operator[](sycl::id<dimensions> index) const;
+This function may only be called from within a command.
 
-  *Atomic variants*
-  atomic<dataT, sycl::access::address_space::global_space> operator[](
-    size_t index) const;
-  atomic<dataT, sycl::access::address_space::global_space> operator[](
-    id<dimensions> index) const;
+.. _tags_buff_accessors:
 
-  *Single dimension in multi-dimensional buffer*
-  __unspecified__ &operator[](size_t index) const;
+=============================
+Buffer command accessors tags
+=============================
 
-Returns reference or value of element in the associated buffer at the
-requested index.
+Some ``accessor`` constructors take a ``TagT`` parameter,
+which is used to deduce template arguments.
+The permissible values for this parameter are listed in table below.
 
-One dimensional buffers are indexed by a data of type
-size_t. Multi-dimensional buffers may be indexed by a data of type
-``id<dimensions>``, or by a sequence of *[]*, 1 per dimension. For
-example ``a[1][2]``.  The operator returns a reference when the
-accessor allows writes, which requires that ``accessMode`` be one of
-``access::mode::write``, ``accessMode == access::mode::read_write``,
-``accessMode == access::mode::discard_write``, or ``accessMode ==
-access::mode::discard_read_write``. The operator returns an atomic if
-the ``accessMode`` is ``access::mode::atomic``.
+.. list-table::
+  :header-rows: 1
 
+  * - Tag value
+    - Access mode
+    - Accessor target
+  * - ``read_write;``
+    - ``access_mode::read_write``
+    - ``target::device``
+  * - ``read_only;``
+    - ``access_mode::read``
+    - ``target::device``
+  * - ``write_only;``
+    - ``access_mode::write``
+    - ``target::device``
+  * - ``read_write_host_task;``
+    - ``access_mode::read_write``
+    - ``target::host_task``
+  * - ``read_only_host_task;``
+    - ``access_mode::read``
+    - ``target::host_task``
+  * - ``write_only_host_task;``
+    - ``access_mode::write``
+    - ``target::host_task``
 
-``get_pointer``
-===============
 
-.. parsed-literal::
+``read-only accessors``
+=======================
 
-  *Available only when:
-   accessTarget == access::target::host_buffer*
+.. list-table::
+  :header-rows: 1
 
-  dataT \*get_pointer() const;
+  * - Data type
+    - Access mode
+  * - not const-qualified
+    - ``access_mode::read``
+  * - const-qualified
+    - ``access_mode::read``
 
-  *Available only when:
-   accessTarget == access::target::global_buffer*
+The specializations of ``accessor`` with
+``target::device or target::host_task`` that are read-only accessors.
```

Comment:
"or" in code

Reply:
fixed https://github.com/imvucic/SYCL_Reference/pull/20/commits/7b6e03dc9cca91697627588f7f7cab406545983a